.github/workflows/build-alpine.yml

```diff
@@ -6,12 +6,12 @@ on:
 
 jobs:
   alpine:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
 
     strategy:
       fail-fast: false
       matrix:
-        target-node: [14, 16, 18, 20]
+        target-node: [14, 16, 18, 20, 22]
```

Comment:
you can drop nodejs 14 everywhere as the plan is to keep latest LTS and latest non-LTS